src/Waargonaut/Decode/Error.hs

```diff
@@ -35,6 +35,7 @@ data DecodeError
   | NumberOutOfBounds JNumber
   | InputOutOfBounds Word64
   | ParseFailed Text
+  | EmptyDecodeFailure
```

Comment:
If you're only going with `Alt` rather than `Alternative`, do we want to remove this guy?